### PR TITLE
Make the tutorial up to date with 1.7.2 on master branch

### LINKED ISSUE #569: In 1.7.2 tutorial, cannot run the python scripts successfully on the notebook UI.
**What deployment mode you are use?  **
Kuberentes.

**What KubeFATE and FATE version you are using?  **
1.7.2
1.4.3

**MUST** Please state the KubeFATE and FATE version you found the issue
1.7.2
1.4.3

**To Reproduce**

Follow the tutorial on minikube

**What happen?**

Cannot run the basic python script on notebook.

**Screenshots**

Screen Shot 2022-03-24 at 4 33 21 PM

**Additional context**

I will fix this issue.

### PR DESCRIPTION
**Pull request by JingChen23**
Fixes  #569

Changes:

1. Optimize the contents of the tutorial.

2. Make sure the python scripts on the notebook can run successfully.
I think there is no need to keep 4 scripts, for the turorial, I think we just need to make sure 3 jobs can passed:
The upload data job, the training job and the predicting job.

To review the doc, check:
https://github.com/JingChen23/KubeFATE/blob/master/docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md

### CODE REVIEW

**Review by LaynePeng — APPROVED** [review]
LGTM. 
Note: 
1. Because this PR is updating documents, so approve to merge into master directly;
2. Need confirm with @owlet42 about removing some outdate tutorials.

File: docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md [hunk 1]
```diff
@@ -189,44 +205,43 @@ PING example.com (192.168.100.123) 56(84) bytes of data.
...
 
-### Verify KubeFATE service
 When `example.com` well set, KubeFATE service version can be shown,
 ```
-kubefate@machine:~/demo$ kubefate version
+kubefate@machine:~/kubefate$ kubefate version
 * kubefate service version=v1.4.1
 * kubefate commandLine version=v1.4.1
 ```
+Note: The `kubefate` CLI can only work in the same directory of config.yaml
```
Comment (owlet42) [inline docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md:214]: The version number here should be v1.4.3.

**Comment by owlet42** [comment]
https://github.com/FederatedAI/KubeFATE/blob/133dff97884ba6484543e63bbb972d41dc342cc7/docker-build/client/pipeline/index.md

This file looks useless, maybe it's better to delete it?

File: docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md (hunk 1 above)
Comment (JingChen23) [reply docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube.md]: Thanks for the finding! Fixed.

**Comment by JingChen23** [comment]
> https://github.com/FederatedAI/KubeFATE/blob/133dff97884ba6484543e63bbb972d41dc342cc7/docker-build/client/pipeline/index.md
> 
> This file looks useless, maybe it's better to delete it?

Nice suggestion!

**Review by owlet42 — APPROVED** [review]
LGTM

**Pull request merged**